### PR TITLE
feature GFM-171 - updated fields on how page

### PR DESCRIPTION
**Pull request by joefitter**
moved account number field to contact by telephone radio option as per service manager request

* updated telephone radio child to a panel containing 2 fields
* moved child field from post radio to telephone
* updated field key
* updated key in fields.json and step config

### CODE REVIEW

File: apps/gro/fields/how.js [hunk 1]
```diff
@@ -13,17 +13,20 @@ module.exports = {
...
+                  {{#input-text}}telephone-toggle-text{{/input-text}}
+                  {{#input-text}}telephone-toggle-text-2{{/input-text}}
+                </div>
+              </div>`
     }, {
       value: 'post',
-      label: 'fields.how-radio.options.post.label',
-      toggle: 'post-toggle-text',
-      child: 'input-text'
+      label: 'fields.how-radio.options.post.label'
```
Comment (JoeChapman) [inline apps/gro/fields/how.js]: don't we get this by default?

Comment (JoeChapman) [inline apps/gro/fields/how.js]: @joefitter ?

Comment (joefitter) [inline apps/gro/fields/how.js]: yeah, I didn't change this line though - would be better as a separate PR to sweep and update all of them IMO

Comment (joefitter) [inline apps/gro/fields/how.js]: I'll submit another PR for these

Comment (joefitter) [inline apps/gro/fields/how.js]: https://jira.digital.homeoffice.gov.uk/browse/GFM-172

**Comment by JoeChapman** [comment]
LGTM
